app/src/main/java/com/kabouzeid/gramophone/ui/fragments/player/card/CardPlayerFragment.java

```diff
@@ -333,6 +324,7 @@ protected void onPostExecute(String lyrics) {
                     }
                 } else {
                     lyricsInfo = new LyricsDialog.LyricInfo(song.title, lyrics);
+                    playerAlbumCoverFragment.setSynchronizedLyrics(AbsSynchronizedLyrics.parse(lyrics));
```

Comment:
Looking at this, I'm wondering if it'd be better to put synchronized lyric info into the existing `LyricInfo` class (and potentially refactor it so it serves both the dialog and synchronized lyrics).